deployment/createMtSvc.sh

```diff
@@ -47,6 +47,7 @@ az keyvault secret set --name token-sign-key  --vault-name $MT_KEYVAULT_NAME --d
 
 # use self signed cert for now
 SSL_PASSWORD=`uuidgen`
+FQDN=`az network traffic-manager profile list --resource-group $COMMON_RESOURCE_GROUP --query [0].dnsConfig.fqdn -o tsv`
```

Comment:
might be beneficial to write this into the cluster info file that is generated?

Reply:
+1

Reply:
where is that file generated?

Reply:
@wesyao are you talking about the ```inception.txt``` that is generated?

Reply:
@billpratt I was referring to `inception.txt`